cli/internal/runsummary/format_text.go

```diff
@@ -42,7 +42,7 @@ func (rsm Meta) FormatAndPrintText(workspaceInfos workspace.Catalog) error {
 	fmt.Fprintln(w1, util.Sprintf("  ${GREY}Global Files\t=\t%d${RESET}", fileCount))
 	fmt.Fprintln(w1, util.Sprintf("  ${GREY}External Dependencies Hash\t=\t%s${RESET}", summary.GlobalHashSummary.RootExternalDepsHash))
 	fmt.Fprintln(w1, util.Sprintf("  ${GREY}Global Cache Key\t=\t%s${RESET}", summary.GlobalHashSummary.GlobalCacheKey))
-	if bytes, err := json.Marshal(summary.GlobalHashSummary.Pipeline); err == nil {
+	if bytes, err := json.Marshal(summary.Pipeline); err == nil {
```

Comment:
We can just remove this, it was purely there for showing global cache inputs. since it's not a global hash input anymore, we don't need to print it. (source: https://github.com/vercel/turbo/pull/3934)

Reply:
I'm okay with dropping it; but I think that having the pipeline as defined in the run summary is super-useful so you don't have to go back to N `turbo.json`s (including workspace configs) to figure out what happened.

That's why I simply promoted it up a level.

Reply:
But these are not resolved configs are they? They are just the root turbo.json. The resolved configs are already in the output for each task and I agree that is super useful